serializers/reviewable_akismet_post_serializer.rb

```diff
@@ -4,4 +4,8 @@
 
 class ReviewableAkismetPostSerializer < ReviewableSerializer
   payload_attributes :post_cooked, :external_error
+
+  def created_from_flag?
```

Comment:
The only use for this is to determine whether we display the context question:

> Is this X Y?

I think it's not a stretch to say that Akismet is flagging posts. WDYT?

Reply:
`created_from_flag` sounds weird? When are reviewables not created from flagging?

Reply:
There are some settings, IIRC. E.g. where new user first post needs review.